thread/src/main/kotlin/io/github/paulgriffith/kindling/thread/ThreadView.kt

```diff
@@ -47,6 +49,15 @@ class ThreadView(
         setSortOrder(ThreadModel[Id], SortOrder.ASCENDING)
     }
```

Comment:
```suggestion
        addHighlighter(ColorHighlighter(
                { _, adapter ->
                    threadDump.deadlocks.contains(adapter.getValue(ThreadModel.ThreadColumns[Id]))
                },
                UIManager.getColor("Actions.Red"),
                null
        ))
    }
```

Also, this whole thing could just be a part of the table's `apply` function, no need for a field + `init` block code.
I don't have this documented anywhere, but I _try_ to stick to a rule of _appearance_ related stuff happens in the component/field initialization, and then the `init` block can just be used to 'wire up' all the event handling and other stuff.

Reply:
Also, `ThreadModel.ThreadColumns[Id]` is type-safe, refactoring-safe, and a bit faster than the `findColumn` approach